src/vl-map-delete-action.js

```diff
@@ -37,10 +37,33 @@ export class VlMapDeleteAction extends VlMapLayerAction {
     }
   }
 
+  /**
+   * Zet de functie die wordt uitgevoerd na het uitvoeren van de verwijder actie.
+   * Voor elke geselecteerde feature wordt de reject callback gebruikt worden om de feature te verwijderen of de reject callback zodat de feature niet verwijderd wordt.
+   *
+   * @param {Function} callback functie met volgende argumenten:
+   *                            - {[ol.Feature]} de te verwijderen features
+   *                            - {Function} resolve callback met ol.Feature als argument die verwijderd wordt
+   *                            - {Function} reject callback zonder argument waarbij de highlight verwijderd wordt
+   */
+  onDelete(callback) {
+    this.__onDeleteCallback = callback;
```

Comment:
Eventueel `__onDeleteCallback` hernoemen naar `__callback` analoog aan de andere actions.

Reply:
Is inderdaad meer in de lijn van de vl-map-action __callback, zal het terug aanpassen.